sdk/core/System.Net.ClientModel/tests/client/OpenAIClientTests.cs

```diff
@@ -31,4 +33,16 @@ public void TestClientSync()
 
         Assert.IsTrue(choice.Text.StartsWith("\n\nLife is"));
```

Comment:
@KrzysztofCwalina, does this test still pass with your change?  I don't have it in the CI's right now because I didn't do the work to make it a recorded test, so we still need to run it manually.

Reply:
It works after I fixed the format check bug (above)

Reply:
TY!